features/naam/naam.feature

```diff
@@ -49,86 +49,176 @@
+    | B                       | baron                      | titel               | M                   | man                   |
+    | BS                      | barones                    | titel               | V                   | vrouw                 |
+    | B                       | baron                      | titel               | O                   | onbekend              |
+    | G                       | graaf                      | titel               | M                   | man                   |
+    | GI                      | gravin                     | titel               | V                   | vrouw                 |
+    | GI                      | gravin                     | titel               | O                   | onbekend              |
+    | H                       | hertog                     | titel               | M                   | man                   |
+    | HI                      | hertogin                   | titel               | V                   | vrouw                 |
+    | M                       | markies                    | titel               | M                   | man                   |
+    | MI                      | markiezin                  | titel               | V                   | vrouw                 |
+    | P                       | prins                      | titel               | M                   | man                   |
+    | PS                      | prinses                    | titel               | V                   | vrouw                 |
+    | R                       | ridder                     | titel               | M                   | man                   |
+    | R                       | ridder                     | titel               | V                   | vrouw                 |
+    | JH                      | jonkheer                   | predicaat           | M                   | man                   |
+    | JV                      | jonkvrouw                  | predicaat           | V                   | vrouw                 |
 
-      Abstract Scenario: adellijke titel en predicaat van de bewoner wordt geleverd bij het raadplegen van bewoning met periode
+
+  Regel: Naam bevat diakrieten
+
+    Abstract Scenario: volledige naam van persoon <omschrijving>
       Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
-      | naam                                 | waarde                    |
-      | voornamen (02.10)                    | Robin Sam                 |
-      | voorvoegsel (02.30)                  | van den                   |
-      | geslachtsnaam (02.40)                | Aedel                     |
-      | adellijke titel of predicaat (02.20) | <adellijkeTitelPredicaat> |
-      | geslachtsaanduiding (04.10)          | <geslachtsAanduiding>     |
+      | naam                        | waarde          |
+      | voornamen (diakrieten)      | <voornamen>     |
```

Comment:
Als je dit realistisch wilt maken, moet zowel voornamen (diacrieten) als voornamen (02.10) gevuld worden. De eerste met diacrieten, de volgende zonder diakrieten.
Zelfde voor geslachtsnaam.

Dus als voornamen/geslachtsnaam diakrieten bevat zijn beide gevuld (met verschillende waarden), als deze geen diakrieten bevat is alleen voornamen (02.10)/geslachtsnaam (02.40) gevuld.